File: 백준/Bronze/2920. 음계/음계.js
let input = require('fs').readFileSync('/dev/stdin').toString().trim();

let asc = "1 2 3 4 5 6 7 8";
let des = "8 7 6 5 4 3 2 1";

if(input === asc){
    console.log("ascending");
} else if(input === des){
    console.log("descending");
} else{
    console.log("mixed");
}

File: 백준/Bronze/2920. 음계/음계.py
arr = list(map(int, input().split()))
count = 0
for i in range(len(arr) - 1) :
    if arr[i] < arr[i + 1] :
        count += 1
    else :
        count -= 1
if count == 7 : print('ascending')
elif count == -7 : print('descending')
else : print('mixed')